Validation/RecoTrack/python/TrackingParticleSelectionsForEfficiency_cff.py

```diff
@@ -29,9 +29,16 @@
 def _modifyForPhase1(pset):
     pset.minRapidity = -3
     pset.maxRapidity = 3
-from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+    pset.tip = 2.5 # beampipe is around 2.0, BPIX1 is at 2.9
 phase1Pixel.toModify(generalTpSelectorBlock, _modifyForPhase1)
 phase1Pixel.toModify(TpSelectorForEfficiencyVsEtaBlock, _modifyForPhase1)
-from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
-phase2_tracker.toModify(generalTpSelectorBlock, minRapidity=-4.5, maxRapidity=4.5)
-phase2_tracker.toModify(TpSelectorForEfficiencyVsEtaBlock, minRapidity=-4.5, maxRapidity=4.5)
+
+run3_common.toModify(generalTpSelectorBlock, _modifyForPhase1)
+run3_common.toModify(TpSelectorForEfficiencyVsEtaBlock, _modifyForPhase1)
```

Comment:
Use of `run3_common` is redundant as `phase1Pixel` modifier is included in `Run3` era.

Reply:
thanks @makortel 
I'll delete these lines, then